### DISCUSSION TITLE
No access to Github Campus for students

### DISCUSSION THREAD

**Discussion by shadow-jpg** [community/community#46264] [GitHub Education]
### Select Topic Area

Product Feedback

### Body

I tryied to register as student for campus, but button was grey and unavailble. where was  sentence something like "Hi,name, you already approved your studying in university you doesnt need to aprove it again, renew you campus". I activated grey button by html changing tag, press it thinking it's a bug, but it gave same message as error. after i switch on teacher and it gave me ability to press the button. I wrote at comments " using github for" my issue, but after coming on new page it doesnt gave me upload a photo with what message, can you help me please? Reaply also doesnt work it gives same page.

Your application cannot be reviewed until you fix the following:

    You applied as a teacher but the email address you used appears to be for student use.,If you meant to apply as a student, please cancel your application and reapply [here](https://education.github.com/discount_requests/student_application).

**Comment by jge162** [comment]
When you login in as teacher you don’t see an option to cancel application?

**Reply by jge162** [reply]
You should reach out to [GitHub Education support ](https://support.github.com/request/education?tags=hubberfy_education) chose other since your scenario is different.

**Comment by cehfisher** [comment]
Hi @shadow-jpg - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to a different category? Thanks!
